java/clients/src/main/java/sleeper/clients/status/report/DeadLettersStatusReport.java

```diff
@@ -68,7 +68,8 @@ public void run() {
             }
         });
         printStats(instanceProperties.get(INGEST_JOB_DLQ_URL), "ingest jobs dead-letter", s -> s);
-        printStats(instanceProperties.get(PARTITION_SPLITTING_DLQ_URL), "partition splitting jobs dead-letter", s ->
+        printStats(instanceProperties.get(PARTITION_SPLITTING_JOB_DLQ_URL), "partition splitting jobs dead-letter",
+                s ->
                 new SplitPartitionJobDefinitionSerDe(tablePropertiesProvider).fromJson(s).toString());
```

Comment:
This has formatted strangely, can this be put onto the same line as the parameter for the lambda declaration?

Reply:
Fixed, thanks